lab_3/main.py

```diff
@@ -20,6 +21,31 @@ def tokenize_by_sentence(text: str) -> tuple:
          (('_', 'h', 'e', '_'), ('_', 'i', 's', '_'), ('_', 'h', 'a', 'p', 'p', 'y', '_'))
          )
     """
+    if not isinstance(text, str):
+        return ()
+    patterns = ['ö', 'ü', 'ä', 'ß']
+    replacements = ['oe', 'ue', 'ae', 'ss']
```

Comment:
If you are not going to change them, it is better to use `tuple`s instead of `list`s.

Reply:
done